packages/codegen-ui-react/lib/primitive.ts

```diff
@@ -108,3 +108,20 @@ export const PrimitiveTypeParameter: Partial<
 export function isBuiltInIcon(componentType: string): boolean {
   return iconset.has(componentType);
 }
+
+/*
+ * temporary list of Primitives with a change event. Final implementation will pull from amplify UI
+ */
+export const PrimitivesWithChangeEvent: Set<Primitive> = new Set([
+  Primitive.CheckboxField,
+  Primitive.PasswordField,
+  Primitive.PhoneNumberField,
+  Primitive.Radio,
```

Comment:
Do we need the underlying radio as well?

Reply:
No we don't need it, bad copy paste.